waku.md

```diff
@@ -142,12 +142,13 @@ Messages with unknown codes MUST be ignored without generating any error, for fo
 
 The Waku sub-protocol MUST support the following packet codes:
 
-| Name                       | Int Value |
-|----------------------------|-----------|
-| Status                     |     0     |
-| Messages                   |     1     |
-| PoW Requirement            |     2     |
-| Bloom Filter               |     3     |
+| Name                       |     Int Value |
+| -------------------------- | ------------- |
+| Status                     |     0         |
+| Messages                   |     1         |
+| PoW Requirement            |     2         |
+| Bloom Filter               |     3         |
+| Rate limits                |     20        |
```

Comment:
14 or 20?

Reply:
Or is this some hex decimal mismatch? If so we shuld probably use 0x

Reply:
Its documented as 0x14, so I assumed its 20.

Reply:
@adambabik do you know if its 20 or 14?

Reply:
or @kdeme

Reply:
There's something weird going on, because the mailserver codes are marked as 126 and 127 in both places.

Whenever e.g. EIP627 uses hex, it uses 0x notation https://eips.ethereum.org/EIPS/eip-627

Reply:
so where did we get `0x14` from?

Reply:
good question, asked here as well https://github.com/status-im/specs/pull/60/files#r351099095

I guess it should be 0x14 as packet code and then 20 in both other places? Not sure why we want to refer to it with hex - convention due to other RLPx subprotocols and RLP?

Reply:
I find the double references a bit confusing, with

> All primitive types are RLP encoded. Note that, per RLP specification, integers are encoded starting from 0x00.

In spec I feel like it is clear enough, and then we should be able to get rid of all the random refers inline text ("packet code rationale" etc)

Reply:
Well it seems kinda stupid cause this is the first time in this spec we have referred to a hex.